go.mod

```diff
@@ -39,10 +38,10 @@ require (
 	github.com/pivotal/image-relocation v0.0.0-20191111101224-e94aff6df06c
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v0.0.6
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.3.2
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/viper v1.4.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
cnab-go pulls in a higher version of testify and packr. The packr bump pulled in a new dependency (envy), etc... It was a cascade from bumping cnab-go and its dependencies.